src/react/DeviceStatusPage/DeviceLiveEventsCard.tsx

```diff
@@ -0,0 +1,347 @@
+/*
+   This file is part of Astarte.
+
+   Copyright 2020 Ispirata Srl
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import React, { useEffect, useState } from 'react';
+import { Badge, Card } from 'react-bootstrap';
+
+import AstarteClient from 'astarte-client';
+import {
+  AstarteDeviceEvent,
+  AstarteDeviceConnectedEvent,
+  AstarteDeviceDisconnectedEvent,
+  AstarteDeviceIncomingDataEvent,
+} from 'astarte-client';
+import FullHeightCard from '../components/FullHeightCard';
+
+interface SystemEvent {
+  level: 'error' | 'info';
+  message: string;
+  timestamp: number;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function isSystemEvent(arg: any): arg is SystemEvent {
+  return arg && arg.level && (arg.level === 'error' || arg.level === 'info');
+}
+
+type RenderableEvent = SystemEvent | AstarteDeviceEvent;
+
+interface WatchDeviceEventsProps {
+  astarte: AstarteClient;
+  deviceId: string;
+  onEventReceived: (event: AstarteDeviceEvent) => void;
+  onErrorMessage: (msg: string) => void;
+  onInfoMessage: (msg: string) => void;
+}
```

Comment:
Since `watchDeviceEvents` is not used as a React component, calling these `Params` instead of `Props` could avoid confusion